src/main/scala/is/hail/expr/ir/IR.scala

```diff
@@ -36,6 +36,30 @@ object Literal {
   }
 }
 
+object Zero {
+  def apply(t: Type): IR = {
```

Comment:
can we have have one of these that takes a type and an int, and converts appropriately?